boards/arm/arduino_portenta_h7/arduino_portenta_h7_m7.dts

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright (c) 2022 Benjamin Björnsson <benjamin.bjornsson@gmail.com>.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+#include <st/h7/stm32h747Xi_m7.dtsi>
+#include <st/h7/stm32h747xihx-pinctrl.dtsi>
+#include "arduino_portenta_h7-common.dtsi"
+
+/ {
+	model = "Arduino Portenta H7 board";
+	compatible = "arduino,portenta-h7";
+
+	/* HW resources are split between CM7 and CM4 */
+	chosen {
+		zephyr,console = &usart1;
+		zephyr,bt-uart = &usart1;
```

Comment:
?

Reply:
I'm not sure what you're referring to?

Reply:
`zephyr,bt-uart = &usart1;` That's a BLE specific property